README.md

```diff
@@ -44,6 +44,14 @@ Spyder may also be used as a PyQt4/PySide extension library (module
 `spyderlib`). For example, the Python interactive shell widget used in
 Spyder may be embedded in your own PyQt4/PySide application.
 
+## Running from source
```

Comment:
@ccordoba12, I agree with taking a developer-first stance on a Github readme, as long as we also mention `pip install` somewhere.